### ISSUE TITLE
can you give a example project use in vue?

### ISSUE THREAD

**Issue by sonygod** [jasononeil/webpack-haxe-loader#56]
can you give a example project use webpack-haxe-loader in vue?

**Comment by elsassph** [comment]
Good idea.

Did you try VueHx (https://github.com/DenkiYagi/vuehx) without Webpack first? It doesn't need Webpack at all.

Or do you want to use the official Vue plugin because you'd like to include some Haxe code in a wider Vue project?

**Comment by sonygod** [comment]
I need to use in real vue dev  with  Vue CLI 3.4.1

I 'd like to include some haxe code inside Vue project,

@elsassph

**Comment by elsassph** [comment]
You can use either Webpack Haxe Loader or hxgenjs, but not together.

Also do you want to interop JS-side Vue components and Haxe side Vue components? Or just include some special content using Haxe (considering your Heaps-related question)?

**Comment by sonygod** [comment]
I try to use Heaps engine in Vue-cli, but it's too big to build every time ,so I want to choose hxgenjs to 

reduce js size suit for hot reload.but it's not work

and I want to use your webpack-haxe-loader for debug only in hx files.

**Comment by elsassph** [comment]
Are you trying to use Heaps as renderer for Vue? Or just render a non-Vue piece using Heaps?

In the second case, if you don't want to build it often it's indeed a good idea to just build it separately, and even load it lazily.

**Comment by vrescobar** [comment]
Hi @sonygod I had the same problem and I ended up forking the repo in order to create the simplest webpack loader for haxe. I use personally use for Vue as well so it includes an example of config for it.

https://github.com/vrescobar/webpack-haxe-loader

**Comment by elsassph** [comment]
@vrescobar the changes you have done seem rather significant so you should consider renaming your fork to avoid confusion if it's not meant to be compatible, or work with us to fix your issues while keeping the expected functionality.

**Comment by vrescobar** [comment]
thanks for the suggestion, I ended up renaming to:
https://github.com/vrescobar/webpack-simple-haxe-loader

Actually, I wanted to ask you, why so much work modifying the args and splitting the code? Supposedly haxe will just remove unused code with DCE.

**Comment by elsassph** [comment]
I agree splitting is a relatively complex feature and it should be possible to completely skip it for small bits of Haxe, but it's not about unused code, it's about reducing the initial JS payload. https://webpack.js.org/guides/code-splitting/
